espanol/dragon/dragon.md

```diff
@@ -1,5 +1,5 @@
 Para iniciar la batalla, tenía que conseguir un medio de transporte.
-Y este medio de transporte sería el Dragón Earhart, que compró al más famoso
+Y este medio de transporte sería el Dragón Earhart, que compró al másfamoso 
```

Comment:
@Isaac12x, was it intentional to delete this space or a mistake? If it was intentional, I'll merge the pull request for you, but it looks like it might have been an accident to me.